application/src/main/java/org/togetherjava/tjbot/commands/utils/StringDistances.java

```diff
@@ -80,6 +76,13 @@ public static Collection<String> closeMatches(CharSequence prefix,
         Queue<MatchScore> bestMatches = new PriorityQueue<>();
         bestMatches.addAll(scoredMatches);
 
+        if (prefix.isEmpty()) {
+            return Stream.generate(bestMatches::poll)
+                .limit(limit)
+                .map(MatchScore::candidate)
+                .toList();
+        }
+
```

Comment:
I feel like this feature is better put into `isCloseEnough` to always say `0.0` when the `prefix.length()` is `0`.

Right now, this feels like an useless duplication